Makefile

```diff
@@ -11,91 +11,122 @@ DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(she
 # but for simplicity sake we just make sure they exist in the first one, and
 # then keep using those.
 FIRST_GOPATH      ?= $(firstword $(subst :, ,$(shell go env GOPATH)))
-PROMU             ?= $(FIRST_GOPATH)/bin/promu
 GOIMPORTS         ?= $(FIRST_GOPATH)/bin/goimports
+PROMU             ?= $(FIRST_GOPATH)/bin/promu
 DEP               ?= $(FIRST_GOPATH)/bin/dep
+ERRCHECK          ?= $(FIRST_GOPATH)/bin/errcheck
 
-all: install-tools deps format build
+.PHONY: all
+all: deps format build
```

Comment:
add `errcheck`? I expect all people to run `make` before raising a PR so would catch it before CI.